packages/keyring-controller/CHANGELOG.md

```diff
@@ -7,6 +7,12 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [17.3.0]
+
+### Changed
+
+- Bump `@metamask/message-manager` from `^10.1.1` to `^11.0.0` ([#4805](https://github.com/MetaMask/core/pull/4805))
```

Comment:
Should this be a breaking change as a major version bump? That is, unless this update doesn't impact keyring-controller.

Reply:
I don't know.  It's possible that a major bump in a dependency does not cause a major change in the consuming package.  But I don't have context on either of these packages.

Reply:
@mikesposito Would it be ok to leave this as non-breaking?

Reply:
`KeyringController` uses this package for (typed) message types, so a breaking change in  `@metamask/message-manager` could potentially also be a breaking change of KeyringController since its types are used in a method signature.

Though, in this case, I think that these changes were intended to be functionally transparent - and as this changelog entry states, the interested types should be unchanged: https://github.com/MetaMask/core/pull/4785/files#diff-1cc33d725972cb3de8b40f38c15b6a5e7014c7ca087c1993ca29645b55daf138R15

So it should be safe to leave this as non-breaking